packages/data-lake-api/package.json

```diff
@@ -15,8 +15,11 @@
     "lint": "eslint . --ext .ts",
     "lint:all": "yarn run lint \"src/**/*.{ts,jsx}\"",
     "lint:fix": "yarn lint --fix",
-    "test": "jest --passWithNoTests",
-    "test:coverage": "yarn test --coverage"
+    "test": "yarn workspace @tupaia/data-lake-api check-test-data-lake-exists && DATA_LAKE_DB_NAME=data_lake_test jest --runInBand",
```

Comment:
Like we do for the tests that require the tupaia database, run a check to see if the data-lake database exists